app/routes/AccountDisplay/AccountDisplay.js

```diff
@@ -26,12 +29,14 @@ class AccountDisplay extends Component {
 		this.actions = [
 			{
 				label: "Log out",
-				onPress: this.userLogout,
+				onPress: this.userLogout.bind(this),
```

Comment:
pour l'instant il faut faire comme ça parce qu'on accède encore à des choses comme le declanchement de l'action du router et aussi au AsyncStorage pour effacer le token.
À partir du moment où on aura plus besoin d'acceder à l`AsyncStorage` pour avoir le token et lAction du Router sera declancher dans un autre endroit, on pourra passer direct `this.props.userLogout()`